pkg/commands/compute/deploy.go

```diff
@@ -68,8 +75,10 @@ func NewDeployCommand(parent cmd.Registerer, client api.HTTPClient, globals *con
 	})
 	c.CmdClause.Flag("path", "Path to package").Short('p').StringVar(&c.Path)
 	c.CmdClause.Flag("domain", "The name of the domain associated to the package").StringVar(&c.Domain)
-	c.CmdClause.Flag("backend", "A hostname, IPv4, or IPv6 address for the package backend").StringVar(&c.Backend)
-	c.CmdClause.Flag("backend-port", "A port number for the package backend").UintVar(&c.BackendPort)
+	c.CmdClause.Flag("backend", "A hostname, IPv4, or IPv6 address for the package backend").StringVar(&c.Backend.Address)
+	c.CmdClause.Flag("backend-port", "A port number for the package backend").UintVar(&c.Backend.Port)
+	c.CmdClause.Flag("override-host", "The hostname to override the Host header").StringVar(&c.Backend.OverrideHost)
+	c.CmdClause.Flag("ssl-sni-hostname", "The hostname to use at the start of the TLS handshake").StringVar(&c.Backend.SSLSNIHostname)
```

Comment:
NIT: With new commands I've added I've been sorting the flags into alphabetical order, so might be worth doing that while we're here.

Reply:
Done.